safenode/src/protocol/transfers/online.rs

```diff
@@ -350,3 +400,38 @@ async fn get_fees(dbc_id: DbcId, client: &Client) -> Result<BTreeMap<NodeId, Req
 
     Ok(results)
 }
+
+/// This will encrypt the necessary components of a fee payment,
+/// so that a node can find and verify the fee paid to them, from
+/// within a request to spend a dbc.
+///
+/// Within all the outputs, will be each fee payment to respective node.
+/// We will encrypt the derivation index to each node public addresss, and
+/// the amount to the derived key corresponding to the dbc id of each node fee payment.
+#[allow(clippy::result_large_err)]
+fn fee_ciphers(
+    outputs: &BTreeMap<DbcId, RevealedAmount>,
+    fee_cipher_params: &BTreeMap<NodeId, (RequiredFee, DbcIdSource)>,
```

Comment:
Param feels weird for a function parameter I'd suggest renaming that to `fees_per_node`

Reply:
Yeah, good point. That name was a bit temporary.

Reply:
I wanted something that conveys that these are to be used for creating the ciphers.

Hmm... But I will think of something.